src/assets/i18n/de.json

```diff
@@ -0,0 +1,239 @@
+{
+	"HEADER": {
+		"SEARCH": "Suche nach Name, Kategorie oder Titel",
+		"GET_STARTED": "Sich aufmachen",
+		"ABOUT_US": "Über uns",
+		"LOGIN": "Anmeldung",
+		"SIGNUP": "ANMELDEN",
+		"BLOG": "Blog",
+		"EXPLORE": "Explore",
+		"DASHBOARD": "Dashboard",
+		"SETTINGS": "Settings",
+		"LOGOUT": "Logout"
+	},
+	"FOOTER": {
+		"ABOUT_US": "About Us",
+		"JOBS": "Arbeitsplätze",
+		"PRESS": "Drücken Sie",
+		"STATISTICS": "Statistiken",
+		"NEWSLETTER": "Newsletter",
+		"SUPPORT": "Unterstützung",
+		"PARTNERS": "Partner",
+		"TUTORIAL": "Tutorial",
+		"BLOG": "Blog",
+		"FAQ": "FAQ",
+		"RULEBOOK": "Regelbuch",
+		"EVENTS": "Veranstaltungen",
+		"TRUST": "Vertrauen",
+		"PRIVACY_POLICY": "Privacy Policy",
+		"TERMS_AND_CONDITION": "Term & Condition",
+		"COOKIE_POLICY": "Cookie Policy",
+		"SUBSRCIBE_NOW": "Got it? Subscribe to our newsletter now !",
+		"PROMISE_MSG": "We don’t send spam emails, promise.",
+		"EMAIL_SEND": "Send",
+		"FOLLOW_US": "Follow Us",
+		"EXPLORE_CAMPAIGN": "EXPLORE CAMPAIGN",
+		"START_A_CAMPAIGN":"START A CAMPAIGN"
+	},
+	"HOME": {
+		"BANNER_TITLE": "The #1 online and mobile fundraising platform.",
+		"BANNER_PARA_1": "Trusted by 3,000+ of the world's top nonprofits & social enterprises.",
+		"CREATE_DREAM": "Create your dream",
+		"DANCE": "Dance",
+		"TECHNOLOGY": "Technology",
+		"MUSIC": "Music",
+		"FOOD": "Food",
+		"PHOTOGRAPHY": "Photography",
+		"FILMS": "Films",
+		"GAMES": "Games",
+		"FASHION": "Fashion",
+		"NON PROFIT": "Non Profit",
+		"BROWSE": "Browse",
+		"ALL_CATEGORIES":"All Categories",
+		"TRENDING_PROJECT": "Trending Projects",
+		"VIEW_ALL": "View All",
+		"VIDEO_INTRODUCTION": "Play Video for Our Introduction",
+		"CUSTOMER_FEEDBACK": "What Our Customer Say’s"
+	},
+	"LOGIN_DIALOG": {
+		"LOGIN": "Login",
+		"SIGNUP_TXT": "New to Crowdpouch Sign up Now!",
```

Comment:
@KnyazNiyaz I think you have used the files from the crowd funding project. We need to have those keys only which are used here in this project. Please make sure you make this change before this PR can be merged. cc: @zeus999

Reply:
that's better?